.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - lts/*
+
+jobs:
+  include:
+    - stage: 'Test the server'
+      before_install: cd backend
+      script: npm run lint
```

Comment:
Won't npm install be required?

Reply:
npm ci will be executed by default...Travis does that